tmt/utils.py

```diff
@@ -776,8 +776,8 @@ def verdict(decision, comment=None, good='pass', bad='fail', problem='warn'):
     """
     Return verdict in green or red based on the decision
 
-    0 or False ... good (green)
-    1 or True .... bad (red)
+    0 or False ... bad (red)
+    1 or True .... good (green)
     otherwise .... problem (yellow)
```

Comment:
The original interpretation here was kind-of unix-like:

 * 0 ... no problems (good)
 * 1 ... failed (bad)
 * 2 ... something weird

Having `0` as the only ok status seems more natural to me.

Reply:
I agree, just tried to fix what the discrepancy between what the function does and what it really did

```
  if decision == 0:
   785         text = style(bad, fg='red')
```

Let's do that outside of this PR ....

Reply:
removed from patchset ..